apps/studio/components/interfaces/Settings/Addons/Addons.tsx

```diff
@@ -155,11 +183,23 @@ const Addons = () => {
 
       {isSuccess && (
         <>
+          {selectedProject?.infra_compute_size === 'nano' && subscription?.plan.id !== 'free' && (
+            <ScaffoldContainer className="mt-4">
+              <Alert_Shadcn_ variant="default">
+                <IconInfo strokeWidth={2} />
+                <AlertTitle_Shadcn_>Free compute upgrade to Micro</AlertTitle_Shadcn_>
+                <AlertDescription_Shadcn_>
+                  Paid Plans include a free upgrade to Micro compute. Your project is ready to
+                  upgrade for no additional charges.
```

Comment:
Since technically it's "$10 of Compute Credits to cover one Micro instance or parts of any other [Compute Add-On](https://supabase.com/docs/guides/platform/compute-add-ons)", as per https://supabase.com/docs/guides/platform/org-based-billing#compute-credits, should the wording make it clearer that that there are other options to use the $10 of Compute Credits besides upgrading to Micro compute?

Reply:
nano and micro are both $10 a month, so it's up to the user to do the free upgrade here - we're just trying to advertise the free upgrade here. Were you thinking about adding something along the lines of "

Paid Plans include a free upgrade to Micro compute. Your project is ready to upgrade for no additional charges. Paid plans come with $10 in Compute Credits to cover one Micro instance or parts of any other [Compute Add-on](https://...)?